examples/heartbeats.py

```diff
@@ -0,0 +1,45 @@
+import unicornhathd as unicorn
+import time
+import colorsys
+import numpy
+
```

Comment:
Needs an introduction, in the form:

```suggestion
print("""Unicorn HAD HD: Heart Beats

Description goes here.

Press Ctrl+C to exit!

""")
```

Reply:
Will add a description.